Edit status check (#1836)
neo4j/docs-operations: modules/ROOT/pages/clustering/monitoring/status-check.adoc

:description: This section describes how to monitor a database's availability with the help of the cluster status check procedure.

:page-role: enterprise-edition new-5.24
[[monitoring-replication]]
= Monitor replication status

Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_statusCheck[`dbms.cluster.statusCheck()`] procedure, which can be used to monitor the ability to replicate in clustered databases. 
In most cases this means a clustered database is write available.

> The procedure can also be used to check which members of a clustered database are up-to-date and can participate in a successful replication.

Reviewer: ```suggestion
The procedure identifies which members of a clustered database are up-to-date and can participate in successful replication.
```

> Therefore, it is useful in determining the fault-tolerance of a clustered database.

Reviewer: ```suggestion
Therefore, it is useful in determining the fault tolerance of a clustered database.
```

> A third and final function is to determine which member is the leader of a clustered database within the cluster.

Reviewer: ```suggestion
Additionally, you can use the procedure to identify the leader of a clustered database within the cluster.
```

| address               | String       | The address of the Bolt port for the server.
| replicationSuccessful | Boolean      | Indicates if the server (on which the procedure is run) can replicate a transaction.
| memberStatus          | String       | The status of each primary member.
| recognisedLeader      | String       | The server id of the perceived leader of each primary member.
| recognisedLeaderTerm  | Integer      | The term of the perceived leader of each primary member. 
If the members report different leaders, the one with the highest term should be trusted.
| requester             | Boolean      | Whether a server is the requester or not.
| error                 | String       | Contains the error message if one is present. 
An example of an error is that one or more of the requested databases do not exist on the requester.
|===

=== Possible values of `replicationSuccessful`
* `TRUE` -- if this server managed to replicate the dummy transaction to a majority of cluster members within the given timeout.
* `FALSE` -- if it failed to replicate within the timeout.
The value is the same column-wise.

> A failed replication can either mean a real issue in the cluster (e.g., no leader) or that this server is too far behind in apply and can't replicate.

Author: 'too far behind in applying updates' is also ok
